src/B2_build_movie_db_framework.sql

```diff
@@ -0,0 +1,41 @@
+CREATE TABLE "user" (
+    user_id INTEGER PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    username VARCHAR(50) UNIQUE NOT NULL,
+    password VARCHAR(255) NOT NULL,
+    email VARCHAR(100) UNIQUE NOT NULL
+);
+
+CREATE TABLE search_history (
+    search_id INTEGER PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    user_id INTEGER REFERENCES "user"(user_id) ON DELETE CASCADE,
+    status VARCHAR(50),
+    note TEXT
```

Comment:
`status` and `note` is not the correct values here. the search history should have a `type` and a `query` column that holds the value of the search query

Reply:
Woops, that seems to have been an oversight on my side. I might have thought about the bookmarks when creating this one. I have changed them to match, what we agreed upon.